selection-to-clipboard/README.md

```diff
@@ -0,0 +1,14 @@
+# selection-to-clipboard
+
+## What it does
+
+This extension includes:
+
+* a content script, "content-script.js", that is injected into all pages
+
+The content script listens for text selections in the page it's attached to and copies the text to the clipboard.
```

Comment:
This doesn't explain that it only does this on mouse up, might be worth commenting on that.